package.json

```diff
@@ -17,26 +17,25 @@
   "author": "GitHub",
   "license": "ISC",
   "dependencies": {
-    "@actions/core": "^1.2.2",
-    "@actions/github": "^2.1.1",
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^5.1.1",
     "tree-kill": "^1.2.2",
-    "uuid": "^7.0.1"
+    "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^25.1.1",
-    "@types/nock": "^11.1.0",
-    "@types/node": "^13.5.3",
-    "@types/uuid": "^7.0.0",
-    "@typescript-eslint/eslint-plugin": "^2.18.0",
-    "@typescript-eslint/parser": "^2.18.0",
-    "@zeit/ncc": "^0.21.0",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-prettier": "^3.1.2",
-    "jest": "^25.1.0",
-    "nock": "^11.7.2",
-    "prettier": "^1.19.1",
-    "ts-jest": "^25.1.0",
-    "typescript": "^3.7.5"
+    "@types/jest": "^29.2.0",
+    "@types/node": "^18.11.3",
+    "@types/uuid": "^8.3.4",
+    "@typescript-eslint/eslint-plugin": "^5.40.1",
+    "@typescript-eslint/parser": "^5.40.1",
+    "@vercel/ncc": "^0.34.0",
```

Comment:
Curious why this was added here?

Reply:
You might need to ask the original author :wink: The modules were there from the initial commit. All I did was update them to newer versions.